### ISSUE TITLE
[solved] cocos.tiles traceback 'ImageDataRegion' object has no attribute 'texture'

### ISSUE THREAD

**Issue by ccanepa** [los-cocos/cocos#321]
test_tiles.py tracebacks with:

```python
(py-cocos) E:\dev\cocos2019\test>python test_tiles.py
Traceback (most recent call last):
  File "test_tiles.py", line 65, in <module>
    main()
  File "test_tiles.py", line 43, in main
    test_layer = tiles.load('road-map.xml')['map0']
  File "..\cocos\tiles.py", line 220, in load
    obj = load_tiles(filename)
  File "..\cocos\tiles.py", line 233, in load_tiles
    resource.handle(root)
  File "..\cocos\tiles.py", line 115, in handle
    return self.factories[tag.tag](self, tag)
  File "..\cocos\tiles.py", line 180, in resource_factory
    self.handle(child)
  File "..\cocos\tiles.py", line 115, in handle
    return self.factories[tag.tag](self, tag)
  File "..\cocos\tiles.py", line 183, in requires_factory
    resource = load(tag.get('file'))
  File "..\cocos\tiles.py", line 220, in load
    obj = load_tiles(filename)
  File "..\cocos\tiles.py", line 233, in load_tiles
    resource.handle(root)
  File "..\cocos\tiles.py", line 115, in handle
    return self.factories[tag.tag](self, tag)
  File "..\cocos\tiles.py", line 180, in resource_factory
    self.handle(child)
  File "..\cocos\tiles.py", line 115, in handle
    return self.factories[tag.tag](self, tag)
  File "..\cocos\tiles.py", line 532, in imageatlas_factory
    image.texture.id
AttributeError: 'ImageDataRegion' object has no attribute 'texture'
```

win7 64.bits, python 3.7

pyglet:
commit 58dfdb6314fc900cfbba6355b79bb09e718a41c9 master github
Author: Benjamin Moran
Date:   Fri Aug 9 09:43:50 2019 +0900
	
cocos:
commit 26ef3497df5d2e6e0d9bb7418681fdfa8cd28d0f master
Author: Richard Jones
Date:   Sun Aug 11 13:15:17 2019 +1000

Other test using cocos.tiles have the same defect (test_tmx_autotest.py, etc)

**Comment by ccanepa** [comment]
commit 6a0a20d1c9787a5b4f139d714518b3290b746bd4 
Date:   Sat Aug 17 02:34:59 2019 -0300

    fix (some?) crashes in cocos.tiles, #321

fixes at least some.

TODO:
  - add to compatibilty_notes
  - rerun autotest\recon.py

**Comment by ccanepa** [comment]
No script in test\ show this traceback after the fix.

Keeping open while doing some other tests to keep in mind.

**Comment by ccanepa** [comment]
fixed in cocos 0.6.7, closing
